### DISCUSSION TITLE
Dynamic Configuration Delivery

### DISCUSSION THREAD

**Discussion by sidai** [redis/go-redis#2132] [Ideas]
We would like to check if contribution of implementing dynamic configuration delivery support is welcomed in this repo? 

To be more specific, we need a resizing capability in the client so that we can adjust the pool size on the fly after a Redis server scaling event without application restart. However, this is infeasible in the current implementation since most related configurations are static and declared once only during client initialization. 

Related issue: https://github.com/go-redis/redis/issues/1430

**Comment by sunisdown** [comment]
Hi @vmihailenco @monkey92t    any idea about this?
